File: cadwiki-nuget/cadwiki.AutoCAD2021.Base.Utilities/Shared/Math.vb
Option Strict On
Option Infer Off
Option Explicit On

Public Class MathUtils
    Public Shared Function RadiansToDegree(radians As Double) As Double
        Dim degrees As Double = 180 * (radians / Math.PI)
        Return degrees
    End Function

    Public Shared Function DegreesToRadians(degrees As Double) As Double
        Dim radians As Double = degrees * (Math.PI / 180)
        Return radians
    End Function


    Public Shared Function AreDoublesEqual(num1 As Double, num2 As Double, fuzz As Double) As Boolean

        Dim dRet As Double = num1 - num2
        Dim bool As Boolean

        If (Math.Abs(dRet) < fuzz) Then
            bool = True
        Else
            bool = False
        End If

        Return bool
    End Function
End Class

File: cadwiki-nuget/cadwiki.AutoCAD2021.Base.Utilities/Shared/Math.cs
using System;

namespace cadwiki.AutoCAD2021.Base.Utilities
{

    public class MathUtils
    {
        public static double RadiansToDegree(double radians)
        {
            double degrees = 180d * (radians / Math.PI);
            return degrees;
        }

        public static double DegreesToRadians(double degrees)
        {
            double radians = degrees * (Math.PI / 180d);
            return radians;
        }


        public static bool AreDoublesEqual(double num1, double num2, double fuzz)
        {

            double dRet = num1 - num2;
            bool @bool;

            if (Math.Abs(dRet) < fuzz)
            {
                @bool = true;
            }
            else
            {
                @bool = false;
            }

            return @bool;
        }
    }
}